src/Planners/tests/UnicycleTrajectoryPlannerTest.cpp

```diff
@@ -0,0 +1,101 @@
+/**
+ * @file UnicyclePlannerTest.cpp
+ * @copyright 2020 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the BSD-3-Clause license.
+ */
+#include <catch2/catch_test_macros.hpp>
+
+#include <manif/manif.h>
+
+#include <iDynTree/Model.h>
+#include <iDynTree/ModelLoader.h>
+
+#include <BipedalLocomotion/ParametersHandler/StdImplementation.h>
+#include <BipedalLocomotion/Planners/UnicycleTrajectoryPlanner.h>
+
+#include <fstream>
+
+#include <FolderPath.h>
+
+using namespace BipedalLocomotion::Planners;
+using namespace BipedalLocomotion::ParametersHandler;
+
+void saveData(const BipedalLocomotion::Contacts::ContactPhaseList& contactPhaseList,
+              const std::string& filename)
+{
+    std::ofstream file(filename);
+    if (!file.is_open())
+    {
+        std::cerr << "Error: Unable to open file " << filename << " for writing." << std::endl;
```

Comment:
Can you use log(). This is valid in general for all the prints

Reply:
done in https://github.com/ami-iit/bipedal-locomotion-framework/commit/5142879658b9cb35aeb4c6425be39b1a5dd3f87c